server/tests/e2e/users.e2e-spec.ts

```diff
@@ -291,6 +291,19 @@ describe('Users', () => {
                 token: token
             }));
 
+        it('should not include REPORT_FILED activities', async () => {
+            prisma.activity.create({ data: { type: ActivityTypes.REPORT_FILED, data: 119n, userID: user.id } });
```

Comment:
Missing await! Surprised this passes the validatePaged count tbh...

Reply:
oops fixed!!